### ISSUE TITLE
pip install tasklib seems to get the version wrong.

### ISSUE THREAD

**Issue by sridhars** [robgolding/tasklib#73]
The current version that pip seems to pick up from pypi seems off in a couple of places.
Currently the pyproject.toml file (in the tar.gz from pypi - not on github) has version 1.30 and refers to older versions of pytz and tzlocal.

I find fewer issues when I replace pyproject.toml versions (not in github) with this:
[tool.poetry]
...
version = "2.1.1"
...
[tool.poetry.dependencies]
pytz = "2020.1"
tzlocal = "2.1"
...

A better workaround may be to remove it entirely or use git+https://github.com/robgolding/tasklib but figured you might want to know.

**Comment by robgolding** [comment]
Hey @sridhars, thanks for the report! Turns out that file was included in the distribution by accident, but I _think_ it should not affect the project in any way. Are you experiencing problems when installing the package due to this?

If so I can publish a new version without the `pyproject.toml` in there!

**Comment by sridhars** [comment]
Yes. I tested it with a modified tar.gz file to confirm. Having the file (as it is currently) pins the version of pytz and tzlocal to older versions. Thanks!

**Comment by xerus2000** [comment]
I had the same problem, tasklib installed with version 1.3.0 for me and didn't want to use any newer version - thus I encountered https://github.com/robgolding/tasklib/issues/38

**Comment by xerus2000** [comment]
```python
❯ sudo pip3 install tasklib
Collecting tasklib
  Downloading tasklib-2.1.1.tar.gz (23 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: pytz==2019.3 in /usr/local/lib/python3.8/dist-packages (from tasklib) (2019.3)
Requirement already satisfied: six==1.14.0 in /usr/local/lib/python3.8/dist-packages (from tasklib) (1.14.0)
Requirement already satisfied: tzlocal==2.0.0 in /usr/local/lib/python3.8/dist-packages (from tasklib) (2.0.0)
Building wheels for collected packages: tasklib
  Building wheel for tasklib (PEP 517) ... done
  Created wheel for tasklib: filename=tasklib-1.3.0-py2.py3-none-any.whl size=24177 sha256=47867ec25fd8f5be7a9414b08a2a857635ae08b9c2dd211c76439d7536f9ea82
  Stored in directory: /tmp/pip-ephem-wheel-cache-fkz4qfrg/wheels/8e/f1/e4/366adb442f7ef514874f484fa32435e50b0f6a5ed03f5d81da
Successfully built tasklib
Installing collected packages: tasklib
Successfully installed tasklib-1.3.0
```
wait what?
It collected the 2.1.1 but the result was 1.3.0?

**Comment by xerus2000** [comment]
And https://github.com/robgolding/tasklib/issues/38 stays broken for me...

**Comment by xerus2000** [comment]
Okay, I got #38 wrong - the issue I have is not addressed yet. just successfully installed the latest version via
`pip install git+https://github.com/robgolding/tasklib`
and my actual issue (tag format - https://github.com/tbabej/taskpirate/issues/11) is still not solved

**Comment by mikevink** [comment]
Just ran into the same problem. @robgolding would you accept a pull request to fix the issue (assuming it can be fixed this way) , so we can get a correct version on pypi?

**Comment by robgolding** [comment]
Sorry about this folks, I've released a new version `2.2.0` which resolves the issue: https://pypi.org/project/tasklib/2.2.0/.

```
rob@boron:~/src
$ pip install --no-cache-dir tasklib==2.2.0
Collecting tasklib==2.2.0
  Downloading tasklib-2.2.0.tar.gz (23 kB)
Collecting pytz
  Downloading pytz-2020.1-py2.py3-none-any.whl (510 kB)
     |████████████████████████████████| 510 kB 3.7 MB/s
Collecting tzlocal
  Downloading tzlocal-2.1-py2.py3-none-any.whl (16 kB)
Using legacy 'setup.py install' for tasklib, since package 'wheel' is not installed.
Installing collected packages: pytz, tzlocal, tasklib
    Running setup.py install for tasklib ... done
Successfully installed pytz-2020.1 tasklib-2.2.0 tzlocal-2.1
```
